Tasks/AzureTestPlanV0/automatedTestInvoker.ts

```diff
@@ -0,0 +1,28 @@
+import * as tl from 'azure-pipelines-task-lib/task'
+import { executepythontests } from './Invokers/pythonivoker'
+import { executemaventests } from './Invokers/maveninvoker'
+import { executegradletests } from './Invokers/gradleinvoker'
+
+export async function testInvoker(testsToBeExecuted: string[]) {
+
+    const testLanguage = tl.getInput('testLanguageInput');
```

Comment:
I guess this is a multi select option, we can store this as an array and use for-loop + switch to decide the respective method.

Reply:
updated it, pls check